### DISCUSSION TITLE
(ANSWERED) Simulate sky from a specified location by using build_stereographic_projection(center)

### DISCUSSION THREAD

**Discussion by Space-A** [skyfielders/python-skyfield#560] [Q&A] [answered]
I'm just getting started using Skyfield for my thesis work, the aim is to simulate the night sky as seen from a time and location specified by the user. I then want to add satellite tracklets to the plot. Currently I have started with the Neowise example, but instead used ISS as a first tracklet object. 

I've been experimenting with the _build_stereographic_projection(center)_ but I am confused about how it works and what should be used for _center_. Currently I'm using _center = station.at(t).observe(ISS_orbit)_ where _station_  is the coordinates for my observation location, and _ISS_orbit = earth + ISS_. However, I read on the [website ](https://rhodesmill.org/skyfield/earth-satellites.html) that this is an expensive method for satellites and something like _geocentric = ISS.at(t)_ should be used instead, but I don't understand how this will take my observation location into account though.  Any ideas or possible clarifications that could get me further?

**Comment by brandon-rhodes** [comment, marked as answer]
If you want a quick relative position between your station and the ISS without the expense of the light-travel time correction in `.observe()`, then I think you can try `(ISS_orbit - station).at(t)`. See whether that gives you the relative position you're interested in!

**Reply by Space-A** [reply]
That did the trick! Thank you very much for your quick response.
